src/components/header/examples/header.scss

```diff
@@ -0,0 +1,6 @@
+:host(limel-example-header) {
+    --header-background-color: rgb(var(--contrast-1100));
```

Comment:
With this background color setting it is difficult to know which icon-buttons are `enabled/disabled`

<img width="935" alt="Screenshot 2021-02-02 at 08 56 10" src="https://user-images.githubusercontent.com/22968497/106569204-8a6c7300-6534-11eb-9da7-0d8413582b67.png">

Reply:
fixed